assets/img/logo.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg viewBox="0 0 172 23" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M167.654 3.35l-4.445 12.001-4.423-12.001h-3.934l6.368 15.907h3.774l6.434-15.907h-3.774z" fill="#112E51"/><path fill-rule="evenodd" clip-rule="evenodd" d="M135.747 11.327c0-4.66 3.758-8.09 8.703-8.09 4.957 0 8.709 3.522 8.709 8.09 0 4.57-3.758 8.091-8.709 8.091-4.95 0-8.703-3.43-8.703-8.09zm3.758-.021c-.015 2.727 2.222 4.952 5 4.973a4.99 4.99 0 003.504-1.471 4.808 4.808 0 001.398-3.48 4.807 4.807 0 00-1.396-3.484 4.99 4.99 0 00-3.506-1.474 5.032 5.032 0 00-3.55 1.443 4.851 4.851 0 00-1.45 3.493z" fill="#112E51"/><path d="M125.324 6.349a6.851 6.851 0 014.445 1.774l2.061-2.567a10.31 10.31 0 00-6.506-2.32c-5.049 0-8.841 3.48-8.841 8.091 0 4.612 3.731 8.091 8.638 8.091a12.136 12.136 0 006.593-2.07v-6.84h-6.027v2.81h2.829v2.098a7.44 7.44 0 01-3.17.863 5.082 5.082 0 01-4.521-2.434 4.891 4.891 0 010-5.062 5.082 5.082 0 014.521-2.434h-.022zm-13.736 8.894a2.03 2.03 0 00-1.493.596 1.956 1.956 0 00-.568 1.481 2.014 2.014 0 00.997 1.839 2.094 2.094 0 002.122 0 2.014 2.014 0 00.997-1.839 1.94 1.94 0 00-.56-1.484 2.015 2.015 0 00-1.495-.593zm-10.368-1.666L93.346 3.35h-3.335v15.907h3.494v-10.2l7.852 10.2h3.33V3.35h-3.467v10.227zM80.731 3.35h3.659v15.907h-3.659zM69.088 6.349a6.853 6.853 0 014.445 1.774l2.06-2.567a10.308 10.308 0 00-6.505-2.32c-5.05 0-8.846 3.48-8.846 8.091 0 4.612 3.73 8.091 8.637 8.091a12.137 12.137 0 006.593-2.07v-6.84h-6.021v2.81h2.83v2.098a7.456 7.456 0 01-3.171.863 5.083 5.083 0 01-4.521-2.434 4.89 4.89 0 010-5.062 5.083 5.083 0 014.52-2.434h-.021z" fill="#112E51"/><path fill-rule="evenodd" clip-rule="evenodd" d="M39.357 11.327c0-4.66 3.758-8.09 8.703-8.09 4.956 0 8.704 3.522 8.704 8.09 0 4.57-3.748 8.091-8.704 8.091s-8.703-3.43-8.703-8.09zm3.747 0c-.003 2.72 2.23 4.931 5 4.952a4.99 4.99 0 003.511-1.467 4.807 4.807 0 001.401-3.485 4.807 4.807 0 00-1.407-3.473 4.99 4.99 0 00-3.505-1.462 5.03 5.03 0 00-3.55 1.442 4.846 4.846 0 00-1.45 3.493z" fill="#112E51"/><path d="M29.808 3.35h-3.66v15.907h10.605v-3.134h-6.945V3.35z" fill="#112E51"/><path d="M15.62 0H9.078v22.655c6.209-2.697 8.379-6.193 8.929-7.282.103-.208.155-.437.153-.669V2.492A2.47 2.47 0 0017.417.73 2.562 2.562 0 0015.62 0z" fill="#B51E23"/><path d="M2.538 0C1.136 0 0 1.116 0 2.492v12.212c-.002.232.05.46.154.669.55 1.079 2.747 4.601 8.923 7.282V0H2.538z" fill="#E21D3E"/><path d="M10.687 9.763a2.502 2.502 0 00.841-2.816 2.574 2.574 0 00-2.435-1.702 2.574 2.574 0 00-2.435 1.702A2.502 2.502 0 007.5 9.763a1082.07 1082.07 0 00-.846 4.758c.8.135 1.61.204 2.423.205.824.016 1.647-.038 2.461-.162-.285-1.618-.549-3.204-.851-4.8z" fill="#fff"/></svg>
```

Comment:
should we lint and expect a viewBox attribute on our SVGs to catch this in the future? Or is it only some SVGs?

Reply:
> should we lint and expect a viewBox attribute on our SVGs to catch this in the future? Or is it only some SVGs?

We can / should probably lint this, yes. Or I guess at least verify that an SVG has a `viewBox` attribute. From what I've read, it seems to be pretty widespread. Given the time-crunch, I'll plan to create a ticket for this to follow-up on.

Reply:
>I'll plan to create a ticket for this to follow-up on.

LG-4039